diff -Nru '--exclude=.git' ollama-0.6.0.org/CMakeLists.txt ollama-0.6.0/CMakeLists.txt
--- ollama-0.6.0.org/CMakeLists.txt	2025-03-16 02:04:26.841112227 +0700
+++ ollama-0.6.0/CMakeLists.txt	2025-03-16 02:27:03.907682777 +0700
@@ -24,6 +24,7 @@
 set(GGML_CUDA_PEER_MAX_BATCH_SIZE 128)
 set(GGML_CUDA_GRAPHS ON)
 set(GGML_CUDA_FA ON)
+set(GGML_LTO ON)
 
 if((CMAKE_OSX_ARCHITECTURES AND NOT CMAKE_OSX_ARCHITECTURES MATCHES "arm64")
     OR (NOT CMAKE_OSX_ARCHITECTURES AND NOT CMAKE_SYSTEM_PROCESSOR MATCHES "arm|aarch64|ARM64|ARMv[0-9]+"))
diff -Nru '--exclude=.git' ollama-0.6.0.org/llama/llama.go ollama-0.6.0/llama/llama.go
--- ollama-0.6.0.org/llama/llama.go	2025-03-16 02:06:01.697843452 +0700
+++ ollama-0.6.0/llama/llama.go	2025-03-16 02:27:36.197762336 +0700
@@ -1,8 +1,8 @@
 package llama
 
 /*
-#cgo CFLAGS: -std=c11
-#cgo CXXFLAGS: -std=c++17
+#cgo CFLAGS: -O3 -flto -std=c11
+#cgo CXXFLAGS: -O3 -flto -std=c++17
 #cgo LDFLAGS: -lstdc++fs
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/include
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/common
