diff -Nru '--exclude=.git' ollama-0.17.6.org/CMakeLists.txt ollama-0.17.6/CMakeLists.txt
--- ollama-0.17.6.org/CMakeLists.txt	2026-03-05 03:01:55.259350693 +0700
+++ ollama-0.17.6/CMakeLists.txt	2026-03-05 03:49:42.333606344 +0700
@@ -41,6 +41,7 @@
 set(GGML_CUDA_PEER_MAX_BATCH_SIZE 128)
 set(GGML_CUDA_GRAPHS ON)
 set(GGML_CUDA_FA ON)
+set(GGML_LTO ON)
 set(GGML_CUDA_COMPRESSION_MODE default)
 
 if((CMAKE_OSX_ARCHITECTURES AND NOT CMAKE_OSX_ARCHITECTURES MATCHES "arm64")
diff -Nru '--exclude=.git' ollama-0.17.6.org/llama/llama.go ollama-0.17.6/llama/llama.go
--- ollama-0.17.6.org/llama/llama.go	2026-03-05 03:35:59.108998352 +0700
+++ ollama-0.17.6/llama/llama.go	2026-03-05 03:51:16.058486609 +0700
@@ -1,9 +1,9 @@
 package llama
 
 /*
-#cgo CFLAGS: -std=c11
+#cgo CFLAGS: -O3 -flto -std=c11
 #cgo windows CFLAGS: -Wno-dll-attribute-on-redeclaration
-#cgo CXXFLAGS: -std=c++17
+#cgo CXXFLAGS: -O3 -flto -std=c++17
 #cgo LDFLAGS: -lstdc++fs
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/include
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/common
