diff -Nru '--exclude=.git' ollama-0.5.11.org/llama/llama.go ollama-0.5.11/llama/llama.go
--- ollama-0.5.11.org/llama/llama.go	2025-02-19 01:11:03.805371949 +0700
+++ ollama-0.5.11/llama/llama.go	2025-02-19 01:27:16.297777394 +0700
@@ -3,6 +3,7 @@
 /*
 #cgo CFLAGS: -std=c11
 #cgo CXXFLAGS: -std=c++17
+#cgo LDFLAGS: -lstdc++fs
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/include
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/common
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/examples/llava
