--- ollama-0.9.0/llama//llama.go.orig	2025-06-02 18:32:21.383634946 +0300
+++ ollama-0.9.0/llama//llama.go	2025-06-02 18:32:44.311123567 +0300
@@ -4,6 +4,7 @@
 #cgo CFLAGS: -std=c11
 #cgo windows CFLAGS: -Wno-dll-attribute-on-redeclaration
 #cgo CXXFLAGS: -std=c++17
+#cgo LDFLAGS: -lstdc++fs
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/include
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/common
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/tools/mtmd
