diff -Nru '--exclude=.git' ollama-0.6.8.org/llama/llama.go ollama-0.6.8/llama/llama.go
--- ollama-0.6.8.org/llama/llama.go	2025-05-11 02:46:57.267978491 +0700
+++ ollama-0.6.8/llama/llama.go	2025-05-11 04:09:04.909315384 +0700
@@ -3,6 +3,7 @@
 /*
 #cgo CFLAGS: -std=c11
 #cgo CXXFLAGS: -std=c++17
+#cgo LDFLAGS: -lstdc++fs
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/include
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/common
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/examples/llava
