diff -Nru '--exclude=.git' ollama-0.17.6.org/llama/llama.go ollama-0.17.6/llama/llama.go
--- ollama-0.17.6.org/llama/llama.go	2026-03-05 03:01:55.283350113 +0700
+++ ollama-0.17.6/llama/llama.go	2026-03-05 03:33:57.773844519 +0700
@@ -4,6 +4,7 @@
 #cgo CFLAGS: -std=c11
 #cgo windows CFLAGS: -Wno-dll-attribute-on-redeclaration
 #cgo CXXFLAGS: -std=c++17
+#cgo LDFLAGS: -lstdc++fs
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/include
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/common
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/vendor
