diff -Nru '--exclude=.git' ollama-0.6.0.org/llama/llama.go ollama-0.6.0/llama/llama.go
--- ollama-0.6.0.org/llama/llama.go	2025-03-16 01:12:18.006914048 +0700
+++ ollama-0.6.0/llama/llama.go	2025-03-16 02:05:05.905418914 +0700
@@ -3,6 +3,7 @@
 /*
 #cgo CFLAGS: -std=c11
 #cgo CXXFLAGS: -std=c++17
+#cgo LDFLAGS: -lstdc++fs
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/include
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/common
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/examples/llava
