diff -Nru '--exclude=.git' ollama-0.12.10.org/llama/llama.go ollama-0.12.10/llama/llama.go
--- ollama-0.12.10.org/llama/llama.go	2025-11-11 04:27:15.116380231 +0700
+++ ollama-0.12.10/llama/llama.go	2025-11-11 04:21:05.213281053 +0700
@@ -4,6 +4,7 @@
 #cgo CFLAGS: -std=c11
 #cgo windows CFLAGS: -Wno-dll-attribute-on-redeclaration
 #cgo CXXFLAGS: -std=c++17
+#cgo LDFLAGS: -lstdc++fs
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/include
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/common
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/vendor
