diff -Nru '--exclude=.git' ollama-0.6.5.org/llama/llama.go ollama-0.6.5/llama/llama.go
--- ollama-0.6.5.org/llama/llama.go	2025-04-12 23:31:40.425749809 +0700
+++ ollama-0.6.5/llama/llama.go	2025-04-13 00:32:58.062663192 +0700
@@ -3,6 +3,7 @@
 /*
 #cgo CFLAGS: -std=c11
 #cgo CXXFLAGS: -std=c++17
+#cgo LDFLAGS: -lstdc++fs
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/include
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/common
 #cgo CPPFLAGS: -I${SRCDIR}/llama.cpp/examples/llava
