diff -Nru '--exclude=.git' ollama-0.5.7.org/make/gpu.make ollama-0.5.7/make/gpu.make
--- ollama-0.5.7.org/make/gpu.make	2025-01-24 07:58:27.713511146 +0700
+++ ollama-0.5.7/make/gpu.make	2025-01-24 07:59:39.705875637 +0700
@@ -56,7 +56,7 @@
 	$(CCACHE) $(GPU_COMPILER) -c -Xcompiler -Wno-unused-function $(GPU_COMPILER_CFLAGS) -o $@ $<
 $(BUILD_DIR)/%.$(GPU_RUNNER_NAME).$(OBJ_EXT): %.cpp
 	@-mkdir -p $(dir $@)
-	$(CCACHE) $(GPU_COMPILER) -c -Xcompiler -Wno-unused-function $(GPU_COMPILER_CXXFLAGS) -o $@ $<
+	$(CCACHE) $(GPU_COMPILER) -c -Xcompiler -Wno-unused-function -std=c++17 $(GPU_COMPILER_CXXFLAGS) -o $@ $<
 $(RUNNERS_BUILD_DIR)/$(GPU_RUNNER_NAME)$(GPU_RUNNER_EXTRA_VARIANT)/ollama_llama_server$(EXE_EXT): TARGET_CGO_LDFLAGS = $(CGO_EXTRA_LDFLAGS) -L"$(RUNNERS_BUILD_DIR)/$(GPU_RUNNER_NAME)$(GPU_RUNNER_EXTRA_VARIANT)/"
 $(RUNNERS_BUILD_DIR)/$(GPU_RUNNER_NAME)$(GPU_RUNNER_EXTRA_VARIANT)/ollama_llama_server$(EXE_EXT): $(RUNNERS_BUILD_DIR)/$(GPU_RUNNER_NAME)$(GPU_RUNNER_EXTRA_VARIANT)/$(SHARED_PREFIX)ggml_$(GPU_RUNNER_NAME).$(SHARED_EXT) ./llama/*.go ./llama/runner/*.go $(COMMON_SRCS) $(COMMON_HDRS)
 	@-mkdir -p $(dir $@)
