diff -Nru '--exclude=.git' ollama-0.6.0.org/ml/backend/ggml/ggml/src/ggml-cuda/CMakeLists.txt ollama-0.6.0/ml/backend/ggml/ggml/src/ggml-cuda/CMakeLists.txt
--- ollama-0.6.0.org/ml/backend/ggml/ggml/src/ggml-cuda/CMakeLists.txt	2025-03-16 01:12:18.026914343 +0700
+++ ollama-0.6.0/ml/backend/ggml/ggml/src/ggml-cuda/CMakeLists.txt	2025-03-16 02:07:03.162294567 +0700
@@ -141,7 +141,7 @@
     endif()
 
     if (NOT MSVC)
-        list(APPEND CUDA_CXX_FLAGS -Wno-pedantic)
+        list(APPEND CUDA_CXX_FLAGS -Wno-pedantic -Wno-unused-function)
     endif()
 
     list(JOIN   CUDA_CXX_FLAGS " " CUDA_CXX_FLAGS_JOINED)  # pass host compiler flags as a single argument
