diff -Nru '--exclude=.git' ollama-0.5.11.org/CMakeLists.txt ollama-0.5.11/CMakeLists.txt
--- ollama-0.5.11.org/CMakeLists.txt	2025-02-19 01:11:03.797372328 +0700
+++ ollama-0.5.11/CMakeLists.txt	2025-02-19 01:23:10.380691864 +0700
@@ -83,6 +83,8 @@
         RUNTIME DESTINATION ${OLLAMA_CUDA_INSTALL_DIR} COMPONENT CUDA
         LIBRARY DESTINATION ${OLLAMA_CUDA_INSTALL_DIR} COMPONENT CUDA
     )
+else()
+    message(FATAL_ERROR "CUDA compiler not found! Ensure CUDA is installed.")
 endif()
 
 set(WINDOWS_AMDGPU_TARGETS_EXCLUDE_REGEX "^gfx(906|908|90a):xnack[+-]$"
diff -Nru '--exclude=.git' ollama-0.5.11.org/ml/backend/ggml/ggml/src/CMakeLists.txt ollama-0.5.11/ml/backend/ggml/ggml/src/CMakeLists.txt
--- ollama-0.5.11.org/ml/backend/ggml/ggml/src/CMakeLists.txt	2025-02-19 01:20:59.964811232 +0700
+++ ollama-0.5.11/ml/backend/ggml/ggml/src/CMakeLists.txt	2025-02-19 01:21:37.432716049 +0700
@@ -302,17 +302,17 @@
     ggml_add_cpu_backend_variant_impl("")
 endif()
 
-ggml_add_backend(BLAS)
-ggml_add_backend(CANN)
+# ggml_add_backend(BLAS)
+# ggml_add_backend(CANN)
 ggml_add_backend(CUDA)
-ggml_add_backend(HIP)
-ggml_add_backend(Kompute)
-ggml_add_backend(METAL)
-ggml_add_backend(MUSA)
-ggml_add_backend(RPC)
-ggml_add_backend(SYCL)
-ggml_add_backend(Vulkan)
-ggml_add_backend(OpenCL)
+# ggml_add_backend(HIP)
+# ggml_add_backend(Kompute)
+# ggml_add_backend(METAL)
+# ggml_add_backend(MUSA)
+# ggml_add_backend(RPC)
+# ggml_add_backend(SYCL)
+# ggml_add_backend(Vulkan)
+# ggml_add_backend(OpenCL)
 
 foreach (target ggml-base ggml)
     target_include_directories(${target} PUBLIC    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../include> $<INSTALL_INTERFACE:include>)
