diff -Nru '--exclude=.git' ollama-0.6.5.org/CMakePresets.json ollama-0.6.5/CMakePresets.json
--- ollama-0.6.5.org/CMakePresets.json	2025-04-12 23:31:40.417749849 +0700
+++ ollama-0.6.5/CMakePresets.json	2025-04-13 00:45:10.253922974 +0700
@@ -21,7 +21,7 @@
       "name": "CUDA 11",
       "inherits": [ "CUDA" ],
       "cacheVariables": {
-        "CMAKE_CUDA_ARCHITECTURES": "50;52;53;60;61;70;75;80;86"
+        "CMAKE_CUDA_ARCHITECTURES": "35;37;50;52;53;60;61;62;70;72;75;80;86;87;89;90"
       }
     },
     {
diff -Nru '--exclude=.git' ollama-0.6.5.org/ml/backend/ggml/ggml/src/ggml-cuda/CMakeLists.txt ollama-0.6.5/ml/backend/ggml/ggml/src/ggml-cuda/CMakeLists.txt
--- ollama-0.6.5.org/ml/backend/ggml/ggml/src/ggml-cuda/CMakeLists.txt	2025-04-13 00:43:29.341394540 +0700
+++ ollama-0.6.5/ml/backend/ggml/ggml/src/ggml-cuda/CMakeLists.txt	2025-04-13 00:45:43.906102439 +0700
@@ -100,7 +100,7 @@
 
     set(CUDA_CXX_FLAGS "")
 
-    set(CUDA_FLAGS -use_fast_math)
+    set(CUDA_FLAGS -use_fast_math -Wno-deprecated-gpu-targets)
 
     if (GGML_FATAL_WARNINGS)
         list(APPEND CUDA_FLAGS -Werror all-warnings)
