diff -Nru '--exclude=.git' ollama-0.6.0.org/CMakePresets.json ollama-0.6.0/CMakePresets.json
--- ollama-0.6.0.org/CMakePresets.json	2025-03-16 01:12:17.970913516 +0700
+++ ollama-0.6.0/CMakePresets.json	2025-03-16 02:11:33.212115187 +0700
@@ -21,7 +21,7 @@
       "name": "CUDA 11",
       "inherits": [ "CUDA" ],
       "cacheVariables": {
-        "CMAKE_CUDA_ARCHITECTURES": "50;52;53;60;61;70;75;80;86"
+        "CMAKE_CUDA_ARCHITECTURES": "35;37;50;52;53;60;61;62;70;72;75;80;86;87;89;90"
       }
     },
     {
diff -Nru '--exclude=.git' ollama-0.6.0.org/ml/backend/ggml/ggml/src/ggml-cuda/CMakeLists.txt ollama-0.6.0/ml/backend/ggml/ggml/src/ggml-cuda/CMakeLists.txt
--- ollama-0.6.0.org/ml/backend/ggml/ggml/src/ggml-cuda/CMakeLists.txt	2025-03-16 02:09:07.743163305 +0700
+++ ollama-0.6.0/ml/backend/ggml/ggml/src/ggml-cuda/CMakeLists.txt	2025-03-16 02:13:29.552838809 +0700
@@ -100,7 +100,7 @@
 
     set(CUDA_CXX_FLAGS "")
 
-    set(CUDA_FLAGS -use_fast_math)
+    set(CUDA_FLAGS -use_fast_math -Wno-deprecated-gpu-targets)
 
     if (GGML_FATAL_WARNINGS)
         list(APPEND CUDA_FLAGS -Werror all-warnings)
