diff -Nru '--exclude=.git' ollama-0.17.6.org/ml/backend/ggml/ggml/src/ggml-cuda/CMakeLists.txt ollama-0.17.6/ml/backend/ggml/ggml/src/ggml-cuda/CMakeLists.txt
--- ollama-0.17.6.org/ml/backend/ggml/ggml/src/ggml-cuda/CMakeLists.txt	2026-03-05 03:40:40.219070730 +0700
+++ ollama-0.17.6/ml/backend/ggml/ggml/src/ggml-cuda/CMakeLists.txt	2026-03-05 03:44:37.347431447 +0700
@@ -120,7 +120,7 @@
 
     set(CUDA_CXX_FLAGS "")
 
-    set(CUDA_FLAGS -use_fast_math -extended-lambda)
+    set(CUDA_FLAGS -use_fast_math -extended-lambda -Wno-deprecated-gpu-targets)
 
     if (GGML_CUDA_DEBUG)
         list(APPEND CUDA_FLAGS -lineinfo)
